.github/workflows/test-action.yml

```diff
@@ -12,6 +12,11 @@ jobs:
       # you must check out the repository
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Run Kani with older version
```

Comment:
Nice! One thing you could do is to change the command to actually check if running `kani --version` matches the version you requested.